environments/production/postgresql.yml

```diff
@@ -54,7 +54,10 @@ dbs:
   form_processing:
     proxy:
       host: pgbouncer3
-      pgbouncer_host: pgbouncer3
+      pgbouncer_endpoint: pgbouncer6
+      pgbouncer_hosts:
+        - pgbouncer3
+        - pgbouncer6
```

Comment:
I don't understand what these options are doing. Are they documented anywhere?

Reply:
Great question. These were introduced in #4447, but I didn't document them. I just wrote up some quick docs for them in https://github.com/dimagi/commcare-cloud/pull/4474.